README.md

```diff
@@ -56,12 +59,13 @@ turbiniactl can be used to start the different components, and here is the basic
 usage:
 
 ```
-$ turbiniactl --help
+$ turbiniactl -h
 usage: turbiniactl [-h] [-q] [-v] [-d] [-a] [-c CONFIG_FILE]
                    [-C RECIPE_CONFIG] [-f] [-o OUTPUT_DIR] [-L LOG_FILE]
                    [-r REQUEST_ID] [-R] [-S] [-V] [-D]
-                   [-F FILTER_PATTERNS_FILE] [-j JOBS_ALLOWLIST]
-                   [-J JOBS_DENYLIST] [-p POLL_INTERVAL] [-t TASK] [-w]
+                   [-F FILTER_PATTERNS_FILE] [-Y YARA_RULES_FILE]
+                   [-j JOBS_ALLOWLIST] [-J JOBS_DENYLIST] [-p POLL_INTERVAL]
+                   [-t TASK] [-w]
```

Comment:
Since debug_task is already merged, we should keep the output here up-to-date. Can you please update this part?

Reply:
Yeah, good call.  I must have created this just before I merged that change.  It should be updated now.  PTAL. Thanks!